### ISSUE TITLE
Form field missing a label in audio component

### ISSUE THREAD

**Issue by poojagunturu96** [middlebury/public-campaign-site#9]
Siteimprove report: https://my2.siteimprove.com/Inspector/7333/A11Y/Page?pageId=82612198767&impmd=D97548790CE293BDC08F6D9B84D3E607&conf=a+aa+aria+si&issue=cantTell+failed&wcag=twopointone&decision=dismissed+falsePositive+none#/sia-r8/failed/cXNVMjdnZWo=;M2UwOWZlMTg=
